packages/google_maps_flutter/google_maps_flutter/lib/src/controller.dart

```diff
@@ -18,6 +18,14 @@ class GoogleMapController {
   /// The mapId for this controller
   final int mapId;
 
+  /// List of active stream subscriptions for map events.
+  ///
+  /// This list keeps track of all event subscriptions created for the map,
+  /// including camera movements, marker interactions, and other map events.
+  /// These subscriptions are properly disposed when the controller is disposed.
```

Comment:
"are properly disposed" -> "should be disposed"

Nothing about the existence of the field guarantees that the disposal actually happens, so the comment on the field shouldn't assert it.